build error on rust 1.39.0

```
$ cargo build
   Compiling librocksdb-sys v6.1.2
error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/bindings.rs:243:5
    |
243 |     pub __opaque: [libc::c_char; 56usize],
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[i8; 56]`
    |
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `[i8; 56]`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&[i8; 56]`
    = note: required for the cast to the object type `dyn std::fmt::Debug`

error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/bindings.rs:282:5
    |
282 |     pub __opaque: [libc::c_char; 40usize],
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[i8; 40]`
    |
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `[i8; 40]`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&[i8; 40]`
    = note: required for the cast to the object type `dyn std::fmt::Debug`

error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/bindings.rs:364:5
    |
364 |     pub __opaque: [libc::c_char; 56usize],
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[i8; 56]`
    |
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `[i8; 56]`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&[i8; 56]`
    = note: required for the cast to the object type `dyn std::fmt::Debug`

error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/bindings.rs:487:5
    |
487 |     pub __opaque: [libc::c_char; 192usize],
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[i8; 192]`
    |
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `[i8; 192]`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&[i8; 192]`
    = note: required for the cast to the object type `dyn std::fmt::Debug`

error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/bindings.rs:572:5
    |
572 |     pub __opaque: [libc::c_char; 8176usize],
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[i8; 8176]`
    |
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `[i8; 8176]`
    = note: required because of the requirements on the impl of `std::fmt::Debug` for `&[i8; 8176]`
    = note: required for the cast to the object type `dyn std::fmt::Debug`

error: aborting due to 5 previous errors

For more information about this error, try `rustc --explain E0277`.
The following warnings were emitted during compilation:

warning: couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
warning: set the LLVM_CONFIG_PATH environment variable to a valid `llvm-config` executable
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier.cc:7:
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier.h:26:
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:265:9: warning: explicitly defaulted copy assignment operator is implicitly deleted [-Wdefaulted-function-deleted]
warning:     IO& operator=(const IO&) = default;
warning:         ^
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:269:29: note: copy assignment operator of 'IO' is implicitly deleted because field 'buf_' is of const-qualified type 'rocksdb::CacheWriteBuffer *const'
warning:     CacheWriteBuffer* const buf_ = nullptr;  // buffer to write
warning:                             ^
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier.cc:7:
warning: rocksdb/utilities/persistent_cache/block_cache_tier.h:96:15: warning: explicitly defaulted move assignment operator is implicitly deleted [-Wdefaulted-function-deleted]
warning:     InsertOp& operator=(InsertOp&& rhs) = default;
warning:               ^
warning: rocksdb/utilities/persistent_cache/block_cache_tier.h:103:16: note: move assignment operator of 'InsertOp' is implicitly deleted because field 'signal_' is of const-qualified type 'const bool'
warning:     const bool signal_ = false;  // signal to request processing thread to exit
warning:                ^
warning: 2 warnings generated.
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier_file.cc:7:
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:265:9: warning: explicitly defaulted copy assignment operator is implicitly deleted [-Wdefaulted-function-deleted]
warning:     IO& operator=(const IO&) = default;
warning:         ^
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:269:29: note: copy assignment operator of 'IO' is implicitly deleted because field 'buf_' is of const-qualified type 'rocksdb::CacheWriteBuffer *const'
warning:     CacheWriteBuffer* const buf_ = nullptr;  // buffer to write
warning:                             ^
warning: 1 warning generated.
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier_metadata.cc:7:
warning: In file included from rocksdb/utilities/persistent_cache/block_cache_tier_metadata.h:15:
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:265:9: warning: explicitly defaulted copy assignment operator is implicitly deleted [-Wdefaulted-function-deleted]
warning:     IO& operator=(const IO&) = default;
warning:         ^
warning: rocksdb/utilities/persistent_cache/block_cache_tier_file.h:269:29: note: copy assignment operator of 'IO' is implicitly deleted because field 'buf_' is of const-qualified type 'rocksdb::CacheWriteBuffer *const'
warning:     CacheWriteBuffer* const buf_ = nullptr;  // buffer to write
warning:                             ^
warning: 1 warning generated.
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/librocksdb.a(db_impl_debug.o) has no symbols
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/librocksdb.a(thread_status_util_debug.o) has no symbols
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/librocksdb.a(sync_point_impl.o) has no symbols
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus_project/electrs/target/debug/build/librocksdb-sys-415ad8884295d9ba/out/librocksdb.a(rocks_lua_compaction_filter.o) has no symbols
warning: snappy/snappy-sinksource.cc:39:36: warning: unused parameter 'length' [-Wunused-parameter]
warning: char* Sink::GetAppendBuffer(size_t length, char* scratch) {
warning:                                    ^
warning: snappy/snappy-sinksource.cc:44:14: warning: unused parameter 'min_size' [-Wunused-parameter]
warning:       size_t min_size, size_t desired_size_hint, char* scratch,
warning:              ^
warning: snappy/snappy-sinksource.cc:44:31: warning: unused parameter 'desired_size_hint' [-Wunused-parameter]
warning:       size_t min_size, size_t desired_size_hint, char* scratch,
warning:                               ^
warning: snappy/snappy-sinksource.cc:82:54: warning: unused parameter 'len' [-Wunused-parameter]
warning: char* UncheckedByteArraySink::GetAppendBuffer(size_t len, char* scratch) {
warning:                                                      ^
warning: snappy/snappy-sinksource.cc:82:65: warning: unused parameter 'scratch' [-Wunused-parameter]
warning: char* UncheckedByteArraySink::GetAppendBuffer(size_t len, char* scratch) {
warning:                                                                 ^
warning: snappy/snappy-sinksource.cc:98:14: warning: unused parameter 'min_size' [-Wunused-parameter]
warning:       size_t min_size, size_t desired_size_hint, char* scratch,
warning:              ^
warning: snappy/snappy-sinksource.cc:98:56: warning: unused parameter 'scratch' [-Wunused-parameter]
warning:       size_t min_size, size_t desired_size_hint, char* scratch,
warning:                                                        ^
warning: snappy/snappy-sinksource.cc:99:14: warning: unused parameter 'scratch_size' [-Wunused-parameter]
warning:       size_t scratch_size, size_t* allocated_size) {
warning:              ^
warning: 8 warnings generated.
warning: snappy/snappy.cc:1180:34: warning: unused parameter 'ip' [-Wunused-parameter]
warning:   inline bool Append(const char* ip, size_t len) {
warning:                                  ^
warning: snappy/snappy.cc:1184:41: warning: unused parameter 'ip' [-Wunused-parameter]
warning:   inline bool TryFastAppend(const char* ip, size_t available, size_t length) {
warning:                                         ^
warning: snappy/snappy.cc:1184:52: warning: unused parameter 'available' [-Wunused-parameter]
warning:   inline bool TryFastAppend(const char* ip, size_t available, size_t length) {
warning:                                                    ^
warning: snappy/snappy.cc:1184:70: warning: unused parameter 'length' [-Wunused-parameter]
warning:   inline bool TryFastAppend(const char* ip, size_t available, size_t length) {
warning:                                                                      ^
warning: snappy/snappy.cc:1403:23: warning: comparison of integers of different signs: 'int' and 'std::__1::vector<snappy::SnappySinkAllocator::Datablock, std::__1::allocator<snappy::SnappySinkAllocator::Datablock> >::size_type' (aka 'unsigned long') [-Wsign-compare]
warning:     for (int i = 0; i < blocks_.size(); ++i) {
warning:                     ~ ^ ~~~~~~~~~~~~~~
warning: snappy/snappy.cc:1419:29: warning: unused parameter 'arg' [-Wunused-parameter]
warning:   static void Deleter(void* arg, const char* bytes, size_t size) {
warning:                             ^
warning: snappy/snappy.cc:1419:60: warning: unused parameter 'size' [-Wunused-parameter]
warning:   static void Deleter(void* arg, const char* bytes, size_t size) {
warning:                                                            ^
warning: snappy/snappy.cc:1316:34: warning: comparison of integers of different signs: 'unsigned long' and 'long' [-Wsign-compare]
warning:     if (PREDICT_TRUE(offset - 1u < op_ptr_ - op_base_ && op_end <= op_limit_)) {
warning:                      ~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~
warning: snappy/snappy-stubs-internal.h:80:25: note: expanded from macro 'PREDICT_TRUE'
warning: #define PREDICT_TRUE(x) x
warning:                         ^
warning: snappy/snappy.cc:715:22: note: in instantiation of member function 'snappy::SnappyScatteredWriter<snappy::SnappySinkAllocator>::AppendFromSelf' requested here
warning:         if (!writer->AppendFromSelf(copy_offset + trailer, length)) {
warning:                      ^
warning: snappy/snappy.cc:799:17: note: in instantiation of function template specialization 'snappy::SnappyDecompressor::DecompressAllTags<snappy::SnappyScatteredWriter<snappy::SnappySinkAllocator> >' requested here
warning:   decompressor->DecompressAllTags(writer);
warning:                 ^
warning: snappy/snappy.cc:1460:12: note: in instantiation of function template specialization 'snappy::InternalUncompressAllTags<snappy::SnappyScatteredWriter<snappy::SnappySinkAllocator> >' requested here
warning:     return InternalUncompressAllTags(&decompressor, &writer, uncompressed_len);
warning:            ^
warning: 8 warnings generated.

error: could not compile `librocksdb-sys`.

To learn more, run the command again with --verbose.
```

```
$ rustc --version
rustc 1.39.0 (4560ea788 2019-11-04)
```

Changing rust to 1.38.0, build completed successfully

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,7 +40,7 @@ num_cpus = "1.0"
 openassets-tapyrus = { git = "https://github.com/chaintope/openassets-tapyrus/", tag = "v0.1.0" }
 page_size = "0.4"
 prometheus = "0.5"
-rocksdb = "0.12.1"
+rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", rev = "aa1aa97163b2db54e90e0056b150d5ff11af0068" }
 rust-crypto = "0.2"
 serde = { version = "1.0", features = ["derive"]}
 serde_derive = "1.0"

